src/main/java/nl/tudelft/contextproject/gui/CreateScriptController.java

```diff
@@ -347,60 +598,92 @@ private void setBackButton() {
                 }
             }
 
+            Script bak = new Script(backupList);
```

Comment:
bak stands for backup. Used bak since most backup files use .bak, but will change it to back